### PR TITLE
Release Godot 4.2 beta 5 and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**
To be released tomorrow by midday.

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-2-beta-5.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+excerpt: "More critical fixes are coming your way on the road to Godot 4.2 stable, including smoother transition for 3D projects made in 4.1."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-2-beta-5.webp
+image_caption_title: Liberation
+image_caption_description: A game by Luke Miller
+date: 2023-11-07 12:00:00
+---
+
+Hope you've had a great weekend! We certainly enjoyed hosting a conference for our community — [GodotCon 2023](https://conference.godotengine.org/), with two dozen talks and workshops, giving insights into using the engine, and building it as well. The talks are available online over at our partners at [Chaos Computer Club](https://media.ccc.de/c/godotcon2023), and soon will be uploaded to the [official YouTube channel](https://www.youtube.com/@GodotEngineOfficial/videos) as well.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-2-beta-5.md]: Is "partners" best? It sounds a bit like a business relationship (maybe it is!). If not, maybe "friends"?

File: collections/_article/dev-snapshot-godot-4-2-beta-5.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+But now it's back to business for Godot contributors, as we're entering another week of beta testing for Godot 4.2. Beta 5 addresses a large number of crashes across all engine and editor systems. You should also have less problems when moving Godot 4.1 projects over to 4.2, thanks to a rework to the mesh format conversion tool and new compatibility options for imported glTF scenes.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-2-beta-5.md]:
```suggestion
But now it's back to business for Godot contributors, as we're entering another week of beta testing for Godot 4.2. Beta 5 addresses a large number of crashes across all engine and editor systems. You should also have fewer problems when moving Godot 4.1 projects over to 4.2, thanks to a rework to the mesh format conversion tool and new compatibility options for imported glTF scenes.
```

File: collections/_article/dev-snapshot-godot-4-2-beta-5.md (hunk 1 above)
Comment (YuriSizov) [reply collections/_article/dev-snapshot-godot-4-2-beta-5.md]: They helped organize the event, so yes.

Comment (clayjohn) [inline collections/_article/dev-snapshot-godot-4-2-beta-5.md]:
```suggestion
Hope you've had a great weekend! We certainly enjoyed hosting a conference for our community — [GodotCon 2023](https://conference.godotengine.org/), with two dozen talks and workshops, giving insights into using the engine, and building it as well. The talks are available online over at the site of our partners, the [Chaos Computer Club](https://media.ccc.de/c/godotcon2023), and soon will be uploaded to the [official YouTube channel](https://www.youtube.com/@GodotEngineOfficial/videos) as well.
```
This is just a grammar fix

**Comment by YuriSizov** [comment]
Thanks for reviews!

**Pull request merged**